internal/fusefrontend_reverse/reverse_longnames.go

```diff
@@ -97,5 +97,12 @@ func (rfs *ReverseFS) newNameFile(relPath string) (nodefs.File, fuse.Status) {
 	}
 	content := []byte(rfs.nameTransform.EncryptName(e, dirIV))
 	parentFile := filepath.Join(rfs.args.Cipherdir, pDir)
```

Comment:
Replace this with 

    parentFile := filepath.Join(rfs.args.Cipherdir, pDir, e)